src/olympia/addons/models.py

```diff
@@ -352,6 +352,7 @@ class Addon(OnChangeMixin, ModelBase):
 
     is_experimental = models.BooleanField(default=False,
                                           db_column='experimental')
+    reputation = models.SmallIntegerField(default=0, null=True)
```

Comment:
It's to match the database migration. And I need to allow `NULL`s in the migration in order not to break the site when deploying (otherwise, deploying the migration would break saves on old code)